src/vs/workbench/api/common/extHost.protocol.ts

```diff
@@ -295,7 +296,7 @@ export interface MainThreadTextEditorsShape extends IDisposable {
 }
 
 export interface MainThreadTreeViewsShape extends IDisposable {
-	$registerTreeViewDataProvider(treeViewId: string, options: { showCollapseAll: boolean, canSelectMany: boolean, dragAndDropMimeTypes: string[] | undefined}): Promise<void>;
+	$registerTreeViewDataProvider(treeViewId: string, options: { showCollapseAll: boolean, canSelectMany: boolean, dragAndDropMimeTypes: string[] | undefined }): Promise<void>;
```

Comment:
Different formatters in use 👀 ?

Reply:
I don't even know how that file was committed to begin with, maybe it was with `--no-verify`. But mine matches the formatter we get on save.

Reply:
One option is to run a format checker on PRs to act as a [Poka-yoke](https://en.wikipedia.org/wiki/Poka-yoke) to keep PRs focused only on what changed verses formatting.

Not sure what formatter you are using; prettier has a `--check` option that can be used to verify, and it's simple enough to add to a build pipeline.